featuretools/entityset/entity.py

```diff
@@ -294,6 +294,8 @@ def _create_variables(self, variable_types, index, time_index, secondary_time_in
         variables = []
         variable_types = variable_types.copy() or {}
         string_to_class_map = find_variable_types()
+        # TODO: Remove once Text has been deprecated
```

Comment:
this TODO should be for once Text has been removed

Reply:
Fixed